packages/interaction-kit/src/helpers/arrays.ts

```diff
@@ -0,0 +1,14 @@
+type PrimitiveArray = Array<string | number | boolean | null | undefined>;
+
+// TODO: Rename this? Move this somewhere else? Don't love either the name or the `helpers` directory
```

Comment:
folder name = utils, method name should instead be arraysEqual (since it only compares arrays)

Reply:
I hate `utils` more than I hate `helpers`, and I hate `helpers`